Ng app template: Describe IE 11 support (#1616)
DevExpress/devextreme-documentation: concepts/40 Angular Components/30 Application Template/60 Internet Explorer 11 Support.md

> For the generated application to work in Internet Explorer 11, make the following adjustments:

Reviewer: I would rewrite this a bit:
Specify the following configuration options to make the generated application work in Internet Explorer 11:

Author: These are not really options. How about this?

```suggestion
To make the generated application work in Internet Explorer 11, do the following:
```

Reviewer: LGTM
